examples/pxScene2d/src/rcvrcore/AppSceneContext.js

```diff
@@ -80,6 +81,9 @@ function AppSceneContext(params) {
   this.timers = [];
   this.timerIntervals = [];
   this.webSocketManager = null;
+  this.httpwrap = new http_wrap(this.accessControl);
+  this.httpswrap = new https_wrap(this.accessControl);
+  this.useSparkPermissionForFileAccess = this.innerscene.sparkSetting("useSparkPermissionsForPxFileAccess")
```

Comment:
Please add a check to see if this setting is defined.  Let's change the setting name to "disableFilePermissionCheck".  The default value (if the setting is not found) should be false.  This means that we should honor permissions by default.